corehq/apps/reports/generic.py

```diff
@@ -669,8 +664,12 @@ def export_response(self):
         Returns the tabular export of the data, if available.
         """
         self.is_rendered_as_export = True
+        report_state = self.__getstate__()
+        report_state['request_params']['startdate'] = str(report_state['request']['datespan'].startdate)
+        report_state['request_params']['enddate'] = str(report_state['request']['datespan'].enddate)
+        del report_state['request']['datespan']
```

Comment:
2 things. You could use `datespan = report_state['request'].pop('datespan')` to both remove the 'datespan' key/value from the dictionary and reference the variable when setting 'startdate' and 'enddate'. Also it is more explicit to use `.isoformat()` when converting a date to a string.

```suggestion
         datespan = report_state['request'].pop('datespan')
        report_state['request_params']['startdate'] = datespan.startdate.isoformat()
        report_state['request_params']['enddate'] = datespan.enddate.isoformat()
```